deploy/kubernetes/metricbeat/README.md

```diff
@@ -2,30 +2,145 @@
+$ gcloud container clusters get-credentials <cluster name> --zone <zone> --project <project name>
+```
+
+
+#### Run Metricbeat on Kubernetes
+
+You can check the nodes of the newly created cluster by running the following:
+
+```
+$ kubectl get nodes
+```
+
+Metricbeat gets some metrics from [kube-state-metrics](https://github.com/kubernetes/kube-state-metrics). You can download the project. You can run, for example:
+
+```
+$ kubectl apply -f kube-state-metrics/examples/standard
+```
+
+To deploy Metricbeat to Kubernetes, clone this repository. Don't forget to set the docker image version inside the [metricbeat-daemonset.yaml](metricbeat-daemonset.yaml).
+Remember that you can find [Metricbeat Docker images here](https://www.docker.elastic.co/r/beats/metricbeat).
+After that, run:
+
+```
+$ kubectl apply -f beats/deploy/kubernetes/metricbeat
+```
+
+You can check the pods of your cluster in the namespace `kube-system`. For that, you just run:
+
+```
+$ kubectl get pods -o wide --namespace=kube-system
+```
+
+You should be able to see each pod's node.
+If you are using Linux/Unix/macOS you can request all pods whose names start with metricbeat:
+
+```
+$ kubectl get pods -o wide --namespace=kube-system | grep ^metricbeat
+```
+
+
+#### Visualizing data in Kibana
+
+To visualize your data in Kibana, we will use Elastic Cloud. For that do the following:
+1. [Log in](https://cloud.elastic.co/home) to your Elastic Cloud account.
+2. Select the target deployment.
+3. Navigate to the **Analytics** endpoint and select **Discover**. To see Metricbeat data, make sure the predefined `metricbeat-*` index pattern is selected.
+4. To load a Kibana dashboard, you have to select **Dashboard** on the side menu. For example, you can see data about your Metricbeat DaemonSet on the dashboard *[Metricbeat Kubernetes] DaemonSets*. If, by any chance, you don't have dashboards downloaded in Kibana, you can go [here](https://www.elastic.co/guide/en/beats/metricbeat/current/load-kibana-dashboards.html) to find a solution for that.
+
+If you can't see any data on the index, then you probably haven't set the YAML manifest file as you should. To do that, open the [metricbeat-daemonset.yaml](metricbeat-daemonset.yaml) and update the following:
+
+```YAML
+- name: ELASTIC_CLOUD_ID
+  value: <cloud_id>
+- name: ELASTIC_CLOUD_AUTH
+  value: <username>:<password>
+```
+
+To update the configuration of your Metricbeat DaemonSet you should run:
+```
+$ kubectl apply -f beats/deploy/kubernetes/metricbeat/metricbeat-daemonset.yaml
+```
+
+You should now be able to visualize your data on Kibana.
 
-We use official [Beats Docker images](https://github.com/elastic/beats-docker),
-as they allow external files configuration, a [ConfigMap](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/)
-is used for kubernetes specific settings. Check [metricbeat-daemonset-configmap.yaml](metricbeat-daemonset-configmap.yaml)
-for details.
+#### Kubernetes dashboard
```

Comment:
I find this bit confusing here since this refers to the Out of the Box Kubernetes dashboard and we have a Kubernetes dashboard as part of our K8s integration. On top of that, I am not sure that deploying other manifests is a good idea in a readme like this. Personally I would have added just a link to either https://kubernetes.io/docs/tasks/access-application-cluster/web-ui-dashboard/ or even better suggest to use https://k9scli.io/ or https://k8slens.dev/ in order to avoid adding more resources to the current cluster